bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java

```diff
@@ -108,22 +108,22 @@ public String getSrcset() {
             }
         }
 
+        // in case of dm image and auto smartcrop the srcset needs to generated client side
+        if (dmImage && StringUtils.equals(smartCropRendition, SMART_CROP_AUTO)) {
+            return null;
+        }
+
         int[] widthsArray = getWidths();
         String srcUritemplate = getSrcUriTemplate();
         String[] srcsetArray = new String[widthsArray.length];
         if (widthsArray.length > 0 && srcUritemplate != null) {
-            String srcUriTemplateDecoded = "";
-            try {
-                srcUriTemplateDecoded = URLDecoder.decode(srcUritemplate, StandardCharsets.UTF_8.name());
-            } catch (UnsupportedEncodingException e) {
-                LOGGER.error("Character Decoding failed for " + resource.getPath());
-            }
-            if (srcUriTemplateDecoded.contains("{.width}")) {
+            srcUritemplate = StringUtils.replace(srcUriTemplate, URI_WIDTH_PLACEHOLDER_ENCODED, URI_WIDTH_PLACEHOLDER);
```

Comment:
Not sure why you're skipping decoding the `srcUriTemplate` now, won't this lead to double encoding of `srcsetArray` items?

Reply:
I removed the decoding to fix the issue https://github.com/adobe/aem-core-wcm-components/issues/2140 and to keep the encoding (e.g for whitespaces). Why do think there will be a double encoding?